src/configuration-manager.ts

```diff
@@ -116,6 +116,14 @@ export class ConfigurationManager {
         await this.tspClient?.request(CommandTypes.Configure, args);
     }
 
+    public async configurePlugin(pluginName: string, configuration: unknown): Promise<void> {
+        const args: tsp.ConfigurePluginRequestArguments = {
+            configuration,
+            pluginName,
+        };
+        await this.tspClient?.request(CommandTypes.ConfigurePlugin, args);
```

Comment:
I like compact code so would suggest slight style change:

```suggestion
        await this.tspClient?.request(CommandTypes.ConfigurePlugin, { pluginName, configuration });
```

Reply:
And to be fair, since it becomes a one-liner and it only used from one place, I would just inline it where the command is handled.